src/main/java/org/cardanofoundation/job/schedules/AggregateAnalyticSchedule.java

```diff
@@ -56,7 +63,7 @@ public void refreshAggBalanceAddressTx() {
         System.currentTimeMillis() - currentTime);
   }
 
-  @Scheduled(fixedDelayString = "${jobs.agg-analytic.fixed-delay}")
+  @Scheduled(initialDelay = 10800000, fixedDelayString = "${jobs.agg-analytic.fixed-delay}")
```

Comment:
Refreshing the view will take time as it scans everything. I added an initial delay to avoid scheduler running in the same when we start the application. 

But it seems like it's not the "true" way to handle it. So I just updated it to use a locking mechanism.